.github/workflows/ci.yml

```diff
@@ -11,13 +11,14 @@ jobs:
   build:
     strategy:
       matrix:
+        os: [ubuntu-latest, windows-latest]
         java: 
           - 8
```

Comment:
Consider putting the 3 `java` versions in a list the same way you did `os`, for consistency. `[8, 11, 14]`.  Is there a reason we aren't testing on 17?

Reply:
Sure I can modify this for java, as for why we aren't testing on 17 for 1.3 branch, I am not quite sure. However, java 11 and 17 are used to test for later versions of Job Scheduler